insitu/views/__init__.py

```diff
@@ -11,7 +11,15 @@
 from .crash_me import *
 
 from django.views.generic import TemplateView
+from insitu.models import UserLog
 
 
 class HomeView(TemplateView):
     template_name = 'home.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        user_log = UserLog.objects.filter().order_by('-date')
```

Comment:
.order_by('-date')[:10] i think it can be used directly on the query and won't through an error if there are less?!